packages/itwinui-variables/src/themes/dark.scss

```diff
@@ -5,25 +5,25 @@
 @use '../utils/get-hsl' as *;
 
 // #region SASS variables
-$bg-hue: 203;
-$bg-saturation: 12%; // More saturated than high contrast theme.
+$bg-saturation: 12%;
 $bg-lightness-gap: 3.25%;
 $fg-lightness-gap: 7%;
 // #endregion
 
 // #region Background colors
 // Darkest @ top, lightest @ bottom
-$bg-0: hsl($bg-hue $bg-saturation 0%);
-$bg-1: hsl($bg-hue $bg-saturation ($bg-lightness-gap));
-$bg-2: hsl($bg-hue $bg-saturation ($bg-lightness-gap * 2));
-$bg-3: hsl($bg-hue $bg-saturation ($bg-lightness-gap * 3));
-$bg-4: hsl($bg-hue $bg-saturation ($bg-lightness-gap * 4));
-$bg-5: hsl($bg-hue $bg-saturation ($bg-lightness-gap * 5));
-$bg-6: hsl($bg-hue $bg-saturation ($bg-lightness-gap * 6));
-$bg-7: hsl($bg-hue $bg-saturation ($bg-lightness-gap * 7));
-$bg-8: hsl($bg-hue $bg-saturation ($bg-lightness-gap * 7.5));
-$bg-9: hsl($bg-hue $bg-saturation ($bg-lightness-gap * 8));
-$bg-10: hsl($bg-hue $bg-saturation ($bg-lightness-gap * 9));
+$bg-0: hsl(203 $bg-saturation 0%);
+$bg-100: hsl(203 $bg-saturation ($bg-lightness-gap));
+$bg-200: hsl(203 $bg-saturation ($bg-lightness-gap * 2));
+$bg-300: hsl(203 $bg-saturation ($bg-lightness-gap * 3));
+$bg-400: hsl(203 $bg-saturation ($bg-lightness-gap * 4));
+$bg-500: hsl(203 $bg-saturation ($bg-lightness-gap * 5));
+$bg-600: hsl(203 $bg-saturation ($bg-lightness-gap * 6));
+$bg-700: hsl(203 $bg-saturation ($bg-lightness-gap * 7));
+$bg-750: hsl(203 $bg-saturation ($bg-lightness-gap * 7.5));
+$bg-800: hsl(203 $bg-saturation ($bg-lightness-gap * 8));
+$bg-900: hsl(203 $bg-saturation ($bg-lightness-gap * 9));
+$bg-1000: hsl(203 $bg-saturation ($bg-lightness-gap * 10));
```

Comment:
You added an extra background in this file compared to the dark-hc.scss file

Reply:
Yes, that was intentional.  I noticed that prior to this `--iui-color-background` & `--iui-color-border-subtle` were set to the same color.  `--iui-color-border` was already to set to 900 and we couldn't go any higher, so I introduced `1000`.

That is one benefit to this autogenerated 0-1000 system is we can introduce new #s whenever needed (like this case).